data_handler/scripts/fetch_zklend_specific_token_settings_test.py

```diff
@@ -0,0 +1,165 @@
+
+src = type('src', (), {'blockchain_call': MockBlockchainCall()})
+
+@dataclass
+class TokenSettings:
+    symbol: str
+    # Source: Starkscan, e.g.
+    # https://starkscan.co/token/0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7 for ETH.
+    decimal_factor: decimal.Decimal
+    address: str
+
+TOKEN_SETTINGS: dict[str, TokenSettings] = {
+    "ETH": TokenSettings(
+        symbol="ETH",
+        decimal_factor=decimal.Decimal("1e18"),
+        address="0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
+    ),
+    "wBTC": TokenSettings(
+        symbol="wBTC",
+        decimal_factor=decimal.Decimal("1e8"),
+        address="0x03fe2b97c1fd336e750087d68b9b867997fd64a2661ff3ca5a7c771641e8e7ac",
+    ),
+    "USDC": TokenSettings(
+        symbol="USDC",
+        decimal_factor=decimal.Decimal("1e6"),
+        address="0x053c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8",
+    ),
+    "DAI": TokenSettings(
+        symbol="DAI",
+        decimal_factor=decimal.Decimal("1e18"),
+        address="0x00da114221cb83fa859dbdb4c44beeaa0bb37c7537ad5ae66fe5e0efd20e6eb3",
+    ),
+    "USDT": TokenSettings(
+        symbol="USDT",
+        decimal_factor=decimal.Decimal("1e6"),
+        address="0x068f5c6a61780768455de69077e07e89787839bf8166decfbf92b645209c0fb8",
+    ),
+    "wstETH": TokenSettings(
+        symbol="wstETH",
+        decimal_factor=decimal.Decimal("1e18"),
+        address="0x042b8f0484674ca266ac5d08e4ac6a3fe65bd3129795def2dca5c34ecc5f96d2",
+    ),
+    "LORDS": TokenSettings(
+        symbol="LORDS",
+        decimal_factor=decimal.Decimal("1e18"),
+        address="0x0124aeb495b947201f5fac96fd1138e326ad86195b98df6dec9009158a533b49",
+    ),
+    "STRK": TokenSettings(
+        symbol="STRK",
+        decimal_factor=decimal.Decimal("1e18"),
+        address="0x4718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d",
+    ),
+}
+
+@dataclass
+class ZkLendSpecificTokenSettings:
+    collateral_factor: decimal.Decimal
+    debt_factor: decimal.Decimal
+    liquidation_bonus: decimal.Decimal
+    protocol_token_address: str
+
+@dataclass
+class TokenSettings(ZkLendSpecificTokenSettings):
+    pass
+
+def format_reserve_data_number(string_large_num: str):
+    """
+    Method to convert large string number to decimal.
+    Example:
+    string_large_num = "800000000000000000000000000"
+    return = 0.80
+    """
+    int_large_num = decimal.Decimal(string_large_num)
+    SCALE_FACTOR = decimal.Decimal("1e27")
+    # Example: cast 800000000000000000000000000 to 0.800000000000000000000000000
+    decimal_large_num = int_large_num / SCALE_FACTOR
+    # Example: convert 0.800000000000000000000000000 to 0.80
+    formatted_number = round(decimal_large_num, 2)
+    return formatted_number
+
+
+def get_value_by_name(data_list: list, name: str):
+    return next((item['value'] for item in data_list if item['name'] == name), None)
+
+def get_token_settings(reserve_data: list):
```

Comment:
Why do you copy-paste all the functions when you can import them?

Reply:
Fix it in this commit: 43dc14e674d4c2caa3514fe3d4b8b8e4e3afc400 please let me know.
I reference all the methods, the only method that i keep it is get_token_reserve_data because i have to mock the response